### PR TITLE
Router: add disable namespace check test for host_admitter using unchanged routes

### PR DESCRIPTION
**Pull request by ramr**
@smarterclayton  as requested added an extra specific test for unmodified wildcard and non-wildcard routes for host_admitter - btw, this is also covered albeit a wee bit differently with tests in `TestDisableOwnershipChecksFuzzing`. PTAL Thx

*Edited* also added a check for no rejections on plain routes that the original PR addressed.

### CODE REVIEW

**Comment by ramr** [comment]
@ironcladlou  PTAL if anything else is needed here. Thx

File: pkg/router/controller/host_admitter_test.go [hunk 1]
```diff
@@ -1131,3 +1131,121 @@ func TestHandleNamespaceProcessing(t *testing.T) {
...
+		if tc.Rejected {
+			if err == nil {
+				t.Fatalf("expected error adding route %s, got none", tc.Name)
+			}
+		} else if err != nil {
+			t.Fatalf("unexpected error adding route %s: %v", tc.Name, err)
+		}
+	}
+
+	for i := 0; i < 5; i++ {
```
Comment (ironcladlou) [inline pkg/router/controller/host_admitter_test.go]: What's with the loop?

**Pull request merged**